index.js

```diff
@@ -17,7 +17,7 @@ var Notify = React.createClass({
 	addNotify: function(title, msg, time, theme) {
 		var key = this.key++;
 		this.state[key] = { title: title, msg: msg, time: time, theme: theme };
-		this.setState(this.state);
+		this.forceUpdate();
```

Comment:
I believe you are wrong. He was using this.setState(this.state); to trigger a re-render. That is literally what `this.forceUpdate()` does. 

From the docs:

* you can tell React that it needs to re-run render() by calling forceUpdate().*